Contest_API.md

```diff
@@ -1226,6 +1244,70 @@ Returned data:
 {"id":"nicky","type":"admin"}
 ```
 
+### Account
+
+Provides information on which account the client represents within a given contest, which endpoints properties are visible, and what [capabilities](#capabilities) (typically PUT, POST, PATCH, or DELETE actions) this account has access to or can perform.
+
+The following endpoint is associated with an account:
+
+| Endpoint                 | Mime-type        | Required? | Description
+| :----------------------- | :--------------- | :-------- | :----------
+| `/contests/<id>/account` | application/json | yes       | JSON object representing a single account with properties as defined in the table below.
+
+Properties of account objects:
+
+| Name         | Type                  | Required? | Nullable? | Description
+| :----------- | :-------------------- | :-------- | :-------- | :----------
+| id           | ID                    | yes       | no        | Identifier of the account used to make this request.
+| username     | string                | yes       | no        | The account username.
+| team\_id     | ID                    | depends   | yes       | The team that this account is for. Required iff type is `team`.
```

Comment:
The `required iff` is not correct, since `type` is not a thing (anymore)

Reply:
Removed depends and updated description to match.